### ISSUE TITLE
Question: kanagawa-dragon lualine appearance

### ISSUE THREAD

**Issue by TiLopes** [rebelot/kanagawa.nvim#156]
Should the lualine statusbar look like the picture below? Or is there a problem in my configuration?
![image](https://github.com/rebelot/kanagawa.nvim/assets/106965010/aa9d61b0-6b30-4b0c-9d50-951a59822dfd)

**Comment by jdsutherland** [comment]
I noticed the same issue. It looks like there's a problem with the lualine theme. It ends up using colors from variants other than dragon.

**Comment by jdsutherland** [comment]
I noticed the same issue. It looks like there's a problem with the lualine theme. It ends up using colors from variants other than dragon.

**Comment by jdsutherland** [comment]
You're referring to the colors, right? 
<img width="687" alt="image" src="https://github.com/rebelot/kanagawa.nvim/assets/5385846/a7a4a8a1-dfdf-4ac9-bffd-477ba9a97986">
<img width="687" alt="image" src="https://github.com/rebelot/kanagawa.nvim/assets/5385846/503e8973-72b8-4a8b-9d07-0a3a9264beab">
<img width="686" alt="image" src="https://github.com/rebelot/kanagawa.nvim/assets/5385846/a2b6ce5c-52f6-4c51-a719-3e697c846fa7">
<img width="682" alt="image" src="https://github.com/rebelot/kanagawa.nvim/assets/5385846/70c19516-8e22-43ba-9708-3078fbcc6fb3">

I don't really like the green in insert mode. It's using `diag.ok`. Relatedly, I'm tempted to override the `diag` colors in dragon. To my eye, they don't really fit with dragon but maybe I need to give it more time.

**Comment by TiLopes** [comment]
> You're referring to the colors, right? 
> <img width="687" alt="image" src="https://github.com/rebelot/kanagawa.nvim/assets/5385846/a7a4a8a1-dfdf-4ac9-bffd-477ba9a97986">
> <img width="687" alt="image" src="https://github.com/rebelot/kanagawa.nvim/assets/5385846/503e8973-72b8-4a8b-9d07-0a3a9264beab">
> <img width="686" alt="image" src="https://github.com/rebelot/kanagawa.nvim/assets/5385846/a2b6ce5c-52f6-4c51-a719-3e697c846fa7">
> <img width="682" alt="image" src="https://github.com/rebelot/kanagawa.nvim/assets/5385846/70c19516-8e22-43ba-9708-3078fbcc6fb3">
> 
> I don't love the green in insert mode. It's using `diag.ok`. Relatedly, I'm tempted to override the `diag` colors in dragon. To my eye, they don't fit but maybe I need to give it more time.

Yes. I guess the green is too bright, and a more pastel green would fit better with the rest of the colours. This is my preference, but I don't know where to actually change the colours.

**Comment by jdsutherland** [comment]
<img width="1822" alt="image" src="https://github.com/rebelot/kanagawa.nvim/assets/5385846/6b6840ca-f896-4520-b2d1-cc7b2b5ee7da">
Look at kanagawa/themes.lua, kanagawa/colors.lua, and lualine/themes/kanagawa.lua to see the relevant definitions. It should be possible to customize the lualine colors as mentioned here https://github.com/nvim-lualine/lualine.nvim#customizing-themes

**Comment by rebelot** [comment]
maybe `syn.string`?

**Comment by jdsutherland** [comment]
I ended up with:

```lua
local custom_kanagawa = require'lualine.themes.kanagawa'
local colors = require("kanagawa.colors").setup()
custom_kanagawa.insert.a.bg = colors.palette.dragonGreen2
custom_kanagawa.insert.b.fg = colors.palette.dragonGreen2
custom_kanagawa.inactive.a.bg = colors.theme.ui.bg_m1
custom_kanagawa.inactive.b.bg = colors.theme.ui.bg_m1
custom_kanagawa.inactive.c.bg = colors.theme.ui.bg_m1
```
